services/graphite/service.go

```diff
@@ -21,6 +23,32 @@ const (
 	leaderWaitTimeout = 30 * time.Second
 )
 
+// expvar stats maintained by the Graphite package.
+var epOnce sync.Once
```

Comment:
Yeah, bad names. Slipped into C mode. :-) I will pick better names, as well as using mixed-case, as is Go style.